neps/nep-0001.md

```diff
@@ -100,6 +99,49 @@ the appropriate venue mentioned above. This gives the author a chance to flesh o
 NEP to make properly formatted, of high quality, and to address
 initial concerns about the proposal.
 
+## The Proposal Process
+One of the primary goals of the NEP process is to prevent proposals being stuck in a limbo state of always being in progress, therefore NEP participants shall commit to always either accept or reject a `Draft` proposal.
```

Comment:
I want to play devil's advocate here: I think "stuck in limbo" is a fairly common outcome for many of Rust's RFCs (eg, two of the three RFCs I authored can be described as stuck). This if course is suboptimal, but it seems that the final Result (the Rust language) is doing fine. Stuckness also is a useful signal -- it shows the lack of motivation, implementation capacity, or certainty. 

So I'd say that the process's "primary goal" is to facilitate reliable and proactive protocol evolution, and fast accepting/rejecting is just "one of the goals"